doc/tutorial/ranges/range_solution4.cpp

```diff
@@ -0,0 +1,35 @@
+#include <seqan3/argument_parser/all.hpp>                       // include argument parser
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/container/bitcompressed_vector.hpp>      // include bitcompressed vector
+#include <vector>
+
+using namespace seqan3;
+
+int main(int argc, char ** argv)
+{
+    argument_parser myparser("Vector implementations comparison", argc, argv);
+    int size{};
+    bool use_bitvector{};
+    myparser.add_positional_option(size, "Size of vector");
+    myparser.add_flag(use_bitvector, 'b', "bitvector", "Use bitvector instead of vector");
+    try
+    {
+         myparser.parse();
+    }
+    catch (parser_invalid_argument const & ext)                     // catch user errors
+    {
+        debug_stream << "[Error] " << ext.what() << "\n";
+        return -1;
+    }
+    
+    if(use_bitvector)
+    {
+        bitcompressed_vector<dna4> vector;
+        vector.resize(size, 'A'_dna4);
+    } else
+    {
+        std::vector<dna4> vector;
+        vector.resize(size, 'A'_dna4);
+    }
+    sleep(60);  // sleep for a minute to give time for memory measurement with 'top'
```

Comment:
I would not use sleep here. Can you change the exercise to state you should only use `/usr/bin/time`? It should be available on all operating systems by default